enaml/core/compiler_helpers.py

```diff
@@ -755,5 +778,5 @@ def add_decl_function(node, func, is_override):
     'validate_spec': validate_spec,
     'validate_template': validate_template,
     'validate_unpack_size': validate_unpack_size,
-    'call_func': call_func,
```

Comment:
there's no need to remove this, just add the new helper.

Reply:
I removed it because I was the one to add it in the first place when adding first support for comprehensions.

Reply:
:+1: